Copy and Share Icon with every Quote #26

roiveredritesh opened the pull request.

roiveredritesh committed:
Copy and Share Icon with every Quote #26

JayShukla8 commented:
i don't see the copy share buttons when i run your code 
can you help me see what changes you've made?

roiveredritesh commented:
I have added the icons css in html and icon container css in style.css.
Also added the js code in script.js. You will see the changes in committed files. Also attaching the working screenshot with this comment. Please check.

![image](https://github.com/user-attachments/assets/b057d69b-9c50-4a10-b1bd-118a03961f7e)

JayShukla8 commented:
@all-contributors please add @roiveredritesh for code
